portfolio/src/main/webapp/index.html

```diff
@@ -110,13 +94,22 @@ <h2>
         <div class="desc">Tokyo, Japan</div>
       </div>
 
+      <h3>Why?</h3>
       <p>I would love to travel to these cities (and many more) to experience its culture. I personally believe that my life would not be complete withoug immersing myself in places outside my comfort zone. In addition to my love for traveling, I also enjoy taking photos of people, cities, and nature. Capturing a landscape is one of the most beautiful things. Capturing a busy city is even more captivating. I spent my previous summer in New York City, and some of the photos of the city are some of my favourites.</p>
       <hr>
-      <h3>
-        <cursive>My Hidden Talent</cursive>
-      </h3>
+      <h3>My Hidden Talent</h3>
       <p>Although I write with my left hand, I am actually ambidextrous. I do many activities with both hands and can even write with both hands (not very well, but its legible)!</p>
-      <p><cursive>Connect with me on <a href=https://www.linkedin.com/in/sydney-beasley-6b04b8168>LinkedIn</a> and <a href=https://www.github.com/Sydney-18>GitHub</a></cursive></p>  
+      <hr>
+      <h2>Social Media</h2>
+
+      <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
I would move `<meta>` and `<link>` into the `<head>` element. [See](https://www.w3schools.com/tags/tag_meta.asp). In this way, the `font-awesome.min.css` will stay with the existing `style.css`. By the way, `font-awesome` is an awesome library!